packages/react/src/components/textinput/InputWrapper.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { ChangeEvent } from 'react';
+
+import classNames from '../../utils/classNames';
+import Tooltip from '../tooltip/Tooltip';
+import { IconLock } from '../../icons';
+import styles from './InputWrapper.module.css';
+
+export type InputWrapperProps = {
+  id: string;
+  labelText?: string;
+  labelledBy?: string;
+  alternative?: boolean;
+  className?: string;
+  defaultValue?: string;
+  disabled?: boolean;
+  helperText?: string;
+  hideLabel?: boolean;
+  invalid?: boolean;
+  invalidText?: string;
+  onChange?: (event: ChangeEvent) => void;
```

Comment:
It would be very handy to accept onBlur as prop. For example formik is setting touched value to true after onBlur event. Now it's not possible to show error messages only after touching the field

Reply:
I'm currently updating the TextInput component. With the updated component you're able to pass props to the native input, so this will be possible then.

Reply:
Great! That's event better.